src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest_Steps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+using System.Net;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        private void given_an_empty_time_sync_behaviour_state()
```

Comment:
All methods names should start with a capital letter.

Reply:
for readability in these bdd style tests, some people think it looks cleaner to be different and stay with all lower case